### ISSUE TITLE
Add Support for Rectangle meme

### ISSUE THREAD

**Issue by chadwpetersen** [ibonwe/triangle-memes#14]
Can you please add support to produce rectangle shaped memes. I want to specify a width and a height value as well. So that the meme produced for the rectangle is according to he width and height I supply. Would also be nice to only ask for the width when I select to produce a rectangle only. I don't want to supply a width to the other shapes if they do not need it.

**Comment by HackerGal** [comment]
So the width option should only become available when the user requests a shape that will require a width?

**Comment by Chad-Petersen** [comment]
Yes that would be ideal. To have the ability to only show the the correct questions for the correct shape. If the shape has that the width must equal the height. Then we can just ask for one. But if the nature of the shape is such that the width is not always the same as the height then we should ask the user only the relevant question. Its about making the follow as simple as possible. I think what we should maybe do is to break this down into two separate issues. One to introduce support for a rectangle and have the system then always ask for both. And then at a later stage create a separate issue to handle a way to manage the user input to figure out what the right questions is to ask. What do you think @HackerGal? So maybe for this issue lets just focus on adding support for rectangles and then **always** asking for both height and width.

**Comment by HackerGal** [comment]
I thought this too. It will allow us to use one function for both purposes. Is this acceptable?
